docs/README.md

```diff
@@ -254,6 +255,13 @@ To run all the browser tests:
 sail dusk
 ```
 
+## Staging <a id="staging"></a>
+
+When there are changes that need to be tested before being deployed to production, like an UX decision or testing a new feature, the [Mismatch Finder staging server](https://mismatch-finder-staging.toolforge.org/) can be used. 
+
+To deploy to this server, push your commits to a branch starting with the name `staging/`. 
+For example `staging/my_branch_name`. The changes in the branch will be deployed to the server after all the checks have passed. If it's decided that the contents of the branch will be merged into `main`, a new Pull Request needs to be created with a name that doesn't start with the `staging` prefix and follow the approval process.  
```

Comment:
I think we can actually also merge PRs from a staging branch, so the last sentence is not needed.

Reply:
Last sentence removed. :)